banking_system/banking_system/views.py

```diff
@@ -5,11 +5,16 @@
 
 @login_required
 def homepage(request):
+    context = {}
     email = request.user.email
     user = User.objects.get(email=email)
-    if user.user_type == 'CUSTOMER':
-        return render(request, 'homepage.html')
-    else:
-        if user.user_type == 'T3':
-            return render(request, 't3_employee_home.html')
-        return render(request, 'employee_home.html')
+    context['user_type'] = user.user_type
+    print(user.user_type)
+    return render(request, 'homepage.html', context)
+    # if user.user_type == 'CUSTOMER':
+    #     return render(request, 'homepage.html', context)
+    # elif user.user_type == 'T3':
+    #     return render(request, 't3_employee_home.html')
+    # elif user.user_type == 'T2':
+    #     return render(request, 't2_employee_home.html', context)
+    # return render(request, 'employee_home.html')
```

Comment:
Can we just delete this please? I don't like having commented out old code that we can just git from the git revisions history

Reply:
I didn't delete it so that people who made changes previously can still test it. We can if everyone approves.